src/scss/core/_base.scss

```diff
@@ -1,3 +1,5 @@
+@use '../provider-variables' as *;
```

Comment:
The sass `preprocessorOptions` in the `vite.config.js` file adds `@use 'src/scss/provider-variables.scss' as *;` to the top of all our sass files? Or just the ones that are imported via javascript?

Because that didn't happen for these `/src/scss/core` sass files. I had to add these here for the `$` prefixed variables to work.

Reply:
imports never needed this because they were effectively combined into the provider-core file which would have the include but `@forward` compiles and namespaces an export rather than putting each partial into the same namespace

Reply:
Makes sense 👍